.github/workflows/clippy-check.yml

```
@@ -21,21 +21,17 @@ jobs:
     runs-on: ${{ matrix.platform }}
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: Swatinem/rust-cache@v2
       - name: Install Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: nightly
-          profile: minimal
-          components: clippy
-          default: true
-          override: true
+        run: |
+          rustup set profile minimal
+          rustup toolchain install nightly --component clippy
+          rustup default nightly
+          rustup override set nightly
       - name: Clippy Check
-        uses: actions-rs/clippy-check@v1
+        uses: kristof-mattei/clippy-check@main
```

Sorry, it seems that `actions-rs-plus`'s version of `clippy-check` is more actively maintained, as it's a fork by `kristof-mattei` themself ( See: https://github.com/actions-rs/clippy-check/pull/165#issuecomment-1277061858 ).

Code comparison: https://github.com/kristof-mattei/clippy-check/compare/main...actions-rs-plus:clippy-check:main

```suggestion
        uses: actions-rs-plus/clippy-check@main
```

By the way, there's a new feature which enables working directories choosing, as introduced in PR https://github.com/actions-rs/clippy-check/pull/158. This is merged by `10XGenomics` into their fork of `clippy-check`, which is forked from `actions-rs-plus`.